src/testing/testing_runtime.ts

```diff
@@ -174,10 +174,16 @@ export class TestingRuntimeImpl implements TestingRuntime {
    */
   async destroy() {
     // Only release once.
-    if (this.#isInitialized) {
-      await this.deactivateEventReceivers();
-      await this.#server?.dbosExec.destroy();
-      this.#isInitialized = false;
+    try {
+      if (this.#isInitialized) {
+        await this.deactivateEventReceivers();
+        await this.#server?.dbosExec.destroy();
+        this.#isInitialized = false;
+      }
+    } catch (err) {
+      const e = err as Error;
+      this.#server?.dbosExec?.logger.error(`Error destroying testing runtime: ${e.message}`);
```

Comment:
Minor: I think it'd be better to just `console.log` this. I noticed github actions do their own gobbling of stdout/stderr, and we don't see our logger entries in GHA reports. And the testing runtime is mostly, well, a testing thing..